models/iaf_psc_alpha_multisynapse.h

```diff
@@ -158,6 +162,25 @@ class iaf_psc_alpha_multisynapse : public Archiving_Node
    */
   struct State_
   {
+
+    /**
+     * Enumeration identifying elements in state vector State_::y_.
+     * This enum identifies the elements of the vector. The last element
+     * of this (I_SYN, ) will be repeated n times at the end of the
+     * state vector State_::y with n being the number of synapses.
+     * As a first step this vector will only be used for logging.
```

Comment:
I think the comment would be easier to understand if you gave the expression for the index of the synapses, i.e. (abbreviated), `NUM_FIX_ELEMS + k * NUM_ELEMS_PER_RECEPTOR`.